client/.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+	"env": {
+		"browser": true,
+		"node": true
+	},
+	"extends": [
+		"eslint:recommended",
+		"plugin:react/recommended",
+		"plugin:@typescript-eslint/recommended"
+	],
+	"parser": "@typescript-eslint/parser",
+	"parserOptions": {
+		"ecmaFeatures": {
+			"jsx": true
+		},
+		"ecmaVersion": 12,
+		"sourceType": "module"
+	},
+	"plugins": [
+		"react",
+		"@typescript-eslint"
+	],
+	"rules": {
+		"indent": [
+			"error",
+			"tab"
```

Comment:
Totally opinion based but I think a lot of our code, especially the html rendered in the React components do not look good with tab indentation. What do you think about changing this to:
`"indent": ["error", 2]`